Doc/library/os.rst

```diff
@@ -1429,6 +1443,23 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
    .. versionadded:: 3.3
 
 
+.. function:: posix_openpt(oflag, /)
+
+   Open and return a file descriptor for a master pseudo-terminal device.
+
+   Calls the C standard library function :c:func:`posix_openpt`. The *oflag*
```

Comment:
To avoid the docs warnings, C library functions need to go `nitpick_ignore` in `Doc/conf.py`.
Sorry for not catching this earlier. I pushed this directly to save a review round-trip.